android/app/src/main/kotlin/com/jabook/app/jabook/audio/StuckPlaybackDetector.kt

```
@@ -0,0 +1,201 @@
+    }
+
+    private fun checkPlaybackState() {
+        val state: Int
+        val isPlaying: Boolean
+        val positionMs: Long
+
+        try {
+            state = player.playbackState
+            isPlaying = player.isPlaying
+            positionMs = player.currentPosition
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)